test_run.lua

```diff
@@ -163,15 +163,15 @@ local function wait_fullmesh(self, servers)
                 log.info("%s: bootstrapped", server)
                 break
             end
-            local info = self:eval(server, "box.info")
+            self:eval(server, "box.info")
```

Comment:
To be honest, I don't understand why `test_run:eval() `it is called here.

Reply:
Me too. Just removed line and comment it in a commit message.

Reply:
The line was here from the beginning (48553b239420166e7f7c26dc01f0ec08c36778d6) and it does not look meaningful within the original code too. Looks as a mistake. So I'm okay with removing if tarantool testing works after the change.